tests/test_datasets/test_dataset.py

```diff
@@ -303,7 +303,22 @@ def test_get_sparse_dataset_with_target(self):
         self.assertEqual(len(attribute_names), 20000)
         self.assertNotIn("class", attribute_names)
 
-    def test_get_sparse_dataset(self):
+    def test_get_sparse_dataset_dataframe_with_target(self):
+        X, y, _, attribute_names = self.sparse_dataset.get_data(
+            dataset_format="dataframe", target="class"
+        )
+        self.assertIsInstance(X, pd.DataFrame)
+        self.assertIsInstance(X.dtypes[0], pd.SparseDtype)
+        self.assertEqual(X.shape, (600, 20000))
+
+        self.assertIsInstance(y, pd.Series)
+        self.assertIsInstance(y.dtypes, pd.SparseDtype)
+        self.assertEqual(y.shape, (600,))
+
+        self.assertEqual(len(attribute_names), 20000)
+        self.assertNotIn("class", attribute_names)
+
+    def test_get_sparse_dataset_array(self):
```

Comment:
As we check for the format of `y` in the new test above, maybe we should also check for the format of `y` here?